src/Slim/App.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
```

Comment:
Yeah just for consistency. Feel free to open a strict-all-the-things PR separately